### PR TITLE
Blob Improvements

### LINKED ISSUE #22001: Blob: Lock down the station for ever edition
#### Description of issue
- blob spawns 2 minutes into the round
 - valiant squad of assistants beat it to death with welders
 - `Directive 7-10 to 7-12 Concluded.`
 - one minute passes
 - `Directive 7-10 Initiated`
 - station now permanently locked down

#### Difference between expected and actual behavior
lockdown shouldn't occur when there's no blob

#### Steps to reproduce
good luck. Probably kill a blob within a minute of it spawning?

#### Length of time in which bug has been known to occur
~~#21968 probably.~~ yes surely an unmerged PR will affect this
no idea

#### Server revision
2c36cc95a9a145d86365513d3a6ddd4fb2c0e6b8
[bugfix]

### PR DESCRIPTION
**Pull request by Kurfursten**
General improvements to blob. Undocumented:
* AntagHUD icon for blob carrier before it pops
* Cerebrate now has its own functionally similar role

fixes #22001

🆑
* bugfix: Restrain blobs now applies to your nodes' passive spread as well.
* bugfix: Cerebrates are now properly known as cerebrates, both in telepathic chat and on the roundend report.
* bugfix: Fixes a bug where the station could become permanently locked down if the blob was destroyed before directive 7-10.
* tweak: Players can once again select blob looks.
* rscadd: For roundstart blob only, Clowns may select Clownscape, Chaplains may select Skellington, and Engineering staff may select AME.

### CODE REVIEW

**Comment by spoonylove** [comment]
Chappie getting spooky blob look?

**Comment by Kurfursten** [comment]
Sure if you can get some emojis

**Comment by DeityLink** [comment]
>rscadd: For roundstart blob only, Clowns may select Clownscape and Engineering staff may select AME.

👍 

Surely plasmemes and skellingtons can select skele-blob can't they?

**Comment by Kurfursten** [comment]
I thought about that but I wasn't sure which direction to go with it. The species check probably makes more sense but giving it to a job (like chaplain) seems to be more consistent.

**Comment by Kurfursten** [comment]
Ready to go and tested

File: code/game/gamemodes/blob/blobs/node.dm [hunk 1]
```diff
@@ -46,7 +46,10 @@
 			M.playsound_local(loc, adminblob_beat, 50, 0, null, FALLOFF_SOUNDS, 0)
 
 	for(var/i = 1; i < 8; i += i)
-		Pulse(5, i)
+		if(overmind)
+			Pulse(5, i, overmind)
```
Comment (MadmanMartian) [inline code/game/gamemodes/blob/blobs/node.dm]: Surely you could just have Pulse(5, i, overmind) rather than the checks, as it's okay if overmind is null

**Pull request merged**